packages/terra-folder-tree/src/FolderTree.module.scss

```diff
@@ -7,6 +7,13 @@
     list-style: none;
     margin: 0;
     padding: 0;
+
+    :focus {
+      box-shadow: var(--terra-folder-tree-focus-box-shadow, none);
+      outline: var(--terra-folder-tree-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-folder-tree-focus-outline-offset, -4px);
+      z-index: 1;
```

Comment:
Why do we need to set the z-index to 1?

Reply:
When the tree items are layered on top of each other, the next tree item will be rendered on top of the outline, so this z-index is here to make sure the outline is always visible and on top

Reply:
Why would the tree items be layered on top of each other?  They would only be visible if the folders are expanded.

Reply:
Sorry you're correct, with further testing they do not overlap with one another. This was a remnant from the previous focus indicator story I was working on